homeassistant/components/alarm_control_panel/device_trigger.py

```diff
@@ -129,14 +138,17 @@ async def async_attach_trigger(
     elif config[CONF_TYPE] == "disarmed":
         from_state = STATE_ALARM_TRIGGERED
         to_state = STATE_ALARM_DISARMED
+    elif config[CONF_TYPE] == "arming":
+        from_state = STATE_ALARM_DISARMED
+        to_state = STATE_ALARM_ARMING
     elif config[CONF_TYPE] == "armed_home":
-        from_state = STATE_ALARM_PENDING
```

Comment:
@balloob is this alright, or schould both STATE_ALARM_PENDING and STATE_ALARM_ARMING be accepted as the "from_state"?
How can that be done, would this work:
`        from_state = STATE_ALARM_PENDING or STATE_ALARM_ARMING`

Reply:
Let's do both for backwards compatibility.